executor/executor_write.go

```diff
@@ -662,6 +667,8 @@ func (e *InsertValues) initDefaultValues(row []types.Datum, marked map[int]struc
 				// `insert t (c1) values(1),(2),(3);`
 				// Last insert id will be 1, not 3.
 				variable.GetSessionVars(e.ctx).SetLastInsertID(uint64(recordID))
+				// It's used to retry.
+				rewriteValueCol = c
```

Comment:
AutoIncrement column maybe not PrimaryKey. So should we move this outside the bracket?